source/release-notes.txt

```diff
@@ -4,9 +4,22 @@ Release Notes
 
 .. default-domain:: mongodb
 
-MongoDB Connector for Spark 10.0.0
+MongoDB Connector for Spark 10.1.0
 ----------------------------------
 
-*Released on March 31, 2022*
```

Comment:
[question] I'm not sure of MongoDB standards, should we include release dates?

Reply:
I was taking cues from [our other release notes](https://www.mongodb.com/docs/drivers/node/current/whats-new/#std-label-version-4.13), which don't include dates.